app/services/payment_provider_customers/gocardless_service.rb

```diff
@@ -19,11 +19,23 @@ def create
       )
 
       deliver_success_webhook
+      PaymentProviderCustomers::GocardlessCheckoutUrlJob.perform_later(gocardless_customer)
 
       result.gocardless_customer = gocardless_customer
       result
     end
 
+    def generate_checkout_url
+      billing_request = create_billing_request(gocardless_customer.provider_customer_id)
+      billing_request_flow = create_billing_request_flow(billing_request.id)
+
+      SendWebhookJob.perform_later(
+        :payment_provider_customer_checkout_url,
+        customer,
+        checkout_url: billing_request_flow.authorisation_url
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)